conn.go

```diff
@@ -267,6 +272,11 @@ func (c *conn) buildRequest(ctx context.Context, query string, params []driver.V
 			req.URL.RawQuery = reqQuery.Encode()
 		}
 	}
+	if method == http.MethodGet {
+		reqQuery := req.URL.Query()
+		reqQuery.Add("query", query)
+		req.URL.RawQuery = reqQuery.Encode()
```

Comment:
it would be better to call reqQuery.Encode only one time. previous possible call is in line 272